spec/models/service_template_provision_request_quota_spec.rb

```diff
@@ -104,6 +104,12 @@ def create_service_bundle(user, items, options = {})
         context "active_provisions_by_tenant," do
           let(:quota_method) { :active_provisions_by_tenant }
           it_behaves_like "check_quota"
+
+          it "invalid service_template does not raise error" do
+            requests = load_queue
+            requests.first.update_attributes(:service_template => nil)
```

Comment:
Does `load_queue` only queue `ServiceTemplateProvisionRequest` instances?  In other words, are you sure this first request will by a ServiceTemplateProvisionRequest and therefore expect to have a `service_template` relationship?

Reply:
@gmcculloug Yes, I am sure the first request will be a ServiceTemplateProvisionRequest since load_queue only queues ServiceTemplateProvisionRequest instances.